osu.Game/Overlays/Chat/ChatTabControl.cs

```diff
@@ -37,6 +37,11 @@ public ChatTabControl()
                 TextSize = 20,
                 Padding = new MarginPadding(10),
             });
+
+            AddItem(new Channel

```

Comment:
add the tab directly, rather than using additem and a fake channel. change TabControl if you must (make it possible to add arbitrary `TabItem<T>` from a `protected` context